doc/source/changes/version_0_26.rst.inc

```diff
@@ -254,5 +254,20 @@ Fixes
 
 * fixed array creation with axis(es) given as string containing only one label (axis name and label were inversed).
 
+* fixed extracting LArray object from a CSV or Excel file when second axis is implicit (closes :issue:`372`).
```

Comment:
s/extracting LArray object/reading an array/?
s/second axis is implicit/the columns axis is not explicitly named (via )/?